src/Evolution/Systems/CurvedScalarWave/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ spectre_target_sources(
   Equations.cpp
   TimeDerivative.cpp
   UpwindPenaltyCorrection.cpp
+  VolumeTermsInstantiation.cpp
```

Comment:
@nilsdeppe I'm confused (again) about https://github.com/sxs-collaboration/spectre/pull/2601#discussion_r571150274. Don't be mad at me for asking this again, just trying to understand :) Should the `partial_derivatives` template instantiations in `CurvedScalarWave/TimeDerivative.cpp` just be deleted? The reason I'm confused is that I think the scalar wave system doesn't do those instantiations, so I don't understand how they can still be needed.

Edit: Actually looking at this in a bit more detail it seems `Evolution/Initialization/Evolution.hpp` includes `Divergence.tpp` and through that it also includes `PartialDerivatives.tpp`, so it might just be generating these template instantiations for `EvolveScalarWave` when compiling the executable. Not sure if that's intended, but it would explain why the executable compiles :)